spec/models/picture_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+describe Picture, type: :model do
+  it { should have_attached_file(:image) }
+
+  it { should validate_attachment_presence(:image) }
+
+  it { should validate_attachment_content_type(:image).
+              allowing('image/png', 'image/gif').
```

Comment:
Style/MultilineMethodCallIndentation: Use 2 (not 12) spaces for indenting an expression spanning multiple lines.